numba/cuda/cudadrv/nvvm.py

```diff
@@ -274,21 +274,27 @@ def get_log(self):
 
 
 try:
-    NVVM_VERSION = NVVM().get_version()
+    from numba.cuda.cudadrv.runtime import runtime
+    cudart_version_major = runtime.get_version()[0]
 except:
-    # the CUDA driver may not be present
-    NVVM_VERSION = (0, 0)
+    # The CUDA Runtime may not be present
+    cudart_version_major = 0
```

Comment:
Wondering if it makes sense to continue without a RT?

Reply:
If there's no runtime you can't use NVVM, but how does "not continuing" differ from "continuing" here?

Reply:
I suppose the question is more, from where does the "this isn't going to work" exception get raised, any reason for it not to be here?

Reply:
This gets run when Numba is imported, so I don't think we want any exceptions raising here. Usually there'd be an exception on an attempt to compile a kernel, because the list of supported compute capabilities will be empty (as set in the following lines).

Reply:
I see, thanks for clarifying, it's a bit strange in that no CC would not be the real problem but a side effect? However, I think fixing/altering is out of scope here. Happy to leave as-is.